java/src/jmri/jmrit/beantable/LogixNGTableTableAction.java

```diff
@@ -45,6 +48,11 @@ public class LogixNGTableTableAction extends AbstractLogixNGTableAction<NamedTab
     ButtonGroup _buttonGroup = new ButtonGroup();
     JTextField _csvFileName = new JTextField(50);
 
+    ButtonGroup _csvGroup = new ButtonGroup();
+    JRadioButton _csvTabbed = new JRadioButton(Bundle.getMessage("LogixNG_CsvType_Tabbed"));
+    JRadioButton _csvRFC = new JRadioButton(Bundle.getMessage("LogixNG_CsvType_RFC"));
```

Comment:
Please change this code to:
```
    JRadioButton _csvTabbed = new JRadioButton(Table.CsvType.TABBED.toString());
    JRadioButton _csvRFC = new JRadioButton(Table.CsvType.COMMA.toString());
```
And please rename _csvRFC to _csvComma here and in similar places.